request_a_govuk_domain/request/admin/views.py

```diff
@@ -45,7 +45,12 @@ def post(self, request):
         if "_confirm" in request.POST:
             try:
                 # send email
-                send_approval_or_rejection_email(request)
+                if os.environ.get("ENVIRONMENT") != "local":
```

Comment:
Should this work with personal stacks?

Reply:
local represents the docker compose environment, on AWS it will behave as intended.